brainstormforce/ultimate-addons-for-gutenberg
# Font Variant Italic not working when Astra Theme is used.

### Description

- Font variant does not work when Astra Theme is used.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Activate Astra.
- Drag and drop Creative Heading, Select Typography to Sort Mili Goudy style Italic.
- Check on the front end it's working or not.

### Checklist:
